src/common-util/src/main/java/ee/ria/xroad/common/conf/globalconf/ConfigurationDirectoryV2.java

```diff
@@ -62,6 +62,9 @@
  */
 @Slf4j
 public class ConfigurationDirectoryV2 implements ConfigurationDirectory {
+    // overrides expires field from metadata
+    // currently used by asicverifier
+    public static OffsetDateTime expiresOverride;
```

Comment:
A global flag feels like a code smell here. Perhaps it should be an option to GlobalConf.reload() to ignore expiration. Or restore the old behavior that ignores expiration if the metadata does not exists.